README.md

```diff
@@ -42,6 +42,7 @@ Currently supported DNS providers:
 - Hurricane Electric DNS
 - INWX
 - Linode
+- LuaDNS
```

Comment:
In the current setup it's necessary to also include the new provider in [documentation/SUMMARY.md](https://github.com/StackExchange/dnscontrol/blob/v3.27.1/documentation/SUMMARY.md?plain=1#L113).

Reply:
Thank you.
Document updated.
For details, please see the commit history.